### PR TITLE
CU-245q3hu - Add menu icon to icon list

### PR DESCRIPTION
**Pull request by SirOibaf**

### CODE REVIEW

**Comment by giardiv** [comment]
Task linked: [CU-245q3hu Git folder in project](https://app.clickup.com/t/245q3hu)

File: src/components/icon/list.tsx [hunk 1]
```diff
@@ -1033,6 +1034,20 @@ export function getIcon(
           />
         </svg>
       );
+    case IconName.menu:
+      return (
+        <svg
+          width="5"
+          height="21"
```
Comment (snqb) [inline src/components/icon/list.tsx]: should it be 5 and 21? It seems to be quite a bit different from other stuff.
@giardiv

Comment (giardiv) [reply src/components/icon/list.tsx]: yes it should be 24px height and 24px width, you might have exported the content of the component and not the whole component
CleanShot 2022-01-24 at 13 08 39@2x
CleanShot 2022-01-24 at 13 08 33@2x

Comment (SirOibaf) [reply src/components/icon/list.tsx]: Honestly I just copied this one: https://github.com/logicalclocks/quartz/blob/dev/src/components/header/menu-button.tsx - but I can fix the sizes

**Comment by SirOibaf** [comment]
Closing this PR as the icon already exists.

**Pull request closed without merging**